test/functional/sqlsrv/skipif_versions_old.inc

```diff
@@ -1,15 +1,21 @@
-<?php
-
-if (! extension_loaded("sqlsrv")) {
-    die("skip extension not loaded");
-}
-
-require_once('MsCommon.inc');
-
-$conn = AE\connect();
-if (! $conn) {
-    echo("Error: could not connect during SKIPIF!");
-} elseif (AE\isColEncrypted() && !AE\isQualified($conn)) {
-    die("skip - AE feature not supported in the current environment.");
-}
+<?php
+
+if (! extension_loaded("sqlsrv")) {
+    die("skip extension not loaded");
+}
+
+require_once('MsCommon.inc');
+
+$conn = AE\connect();
+if (! $conn) {
+    echo("Error: could not connect during SKIPIF!");
+} elseif (AE\isColEncrypted()) {
+    if (!isWindows()) {
+        die( "Skip, AE test on windows only." );
+    }
+
+    if (!AE\isQualified($conn)) {
+        die("skip - AE feature not supported in the current environment.");
+    }
+}

```

Comment:
this file seems to be used to skipif_ae_supported

Reply:
I don't think there's skipif_ae_supported?

Reply:
I mean, that's what it's really doing